requirements/base.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with python 3.8
+# This file is autogenerated by pip-compile with python 3.9
```

Comment:
3.9?

Reply:
Forgot to activate my venv 😓 

Fixed in this PR: https://github.com/Unstructured-IO/unstructured-api-tools/pull/51